docs/git/desktop/lesson.md

```diff
@@ -396,29 +400,42 @@ Celebration!!!
 Make an effort to mark the achievements up to this point. You completed ITD and Fundamentals. Some of you never wrote a line of code before that!
 
 **Session 1: Goals and Expectations**
-The presentation below aims to identify your goal(s) for being on the CYF software development course; to highlight the expectations CodeYourFuture has for its trainees; to introduce the concept of progressing from the Intro to Digital, Fundamentals courses to the software development course.
+
+**Session objective:** The presentation below aims to identify your goal(s) for being on the CYF software development course; to highlight the expectations CodeYourFuture has for its trainees; and to introduce the concept of progressing from the Intro to Digital, Fundamentals courses to the software development course.
 
 [Goals and Expectations](https://docs.google.com/presentation/d/11vimrefwpHfFByUfgPL-jwhZ2KrE0Li80M_O_z2U95I/edit)
 
 Notes for the facilitator: Remember to adapt/update the content in line with your cohort’s needs and arrange timings as needed.
 
 **Session 2: What do developers do?**
-In this section we'll explore how a professional developer behaves and thus how you should behave.
 
-**Exercise (20 min)**
-In small groups. Imagine all the day-to-day activities and behaviours of a professional software developer. Use an online board (Miro, Google Jamboard, etc.). Think about whether you do all those things already. Discuss how to change things so you begin to do more and more of those things. Highlight that practice is the way to develop.
+**Session objective:** In this section we'll explore how a professional developer behaves and thus how you should behave.
+
+:::note Exercise (20 min)
+
+**Exercise objective:** To brainstorm and think about future responsibilities.
+
+In small groups, imagine all the day-to-day activities and behaviours of a professional software developer. Use an online board (Miro, Google Jamboard, etc.). Think about whether you do all those things already. Discuss how to change things so you begin to do more and more of those things. Highlight that practice is the way to develop.
```

Comment:
```suggestion
In small groups, imagine all the day-to-day activities and behaviours of a professional software developer. Use a whiteboard, sticky notes, or an online board (Miro, Google Jamboard, etc.). Think about whether you do all those things already. Discuss how to change things so you begin to do more and more of those things. Highlight that practice is the way to develop.
```
In Fundamentals, we try to provide options for in-person sessions. Could we allow people to use sticky notes instead of online boards or are the online boards necessary to the exercise? (We are working towards making our syllabus available offline in #550 )